README.md

```diff
@@ -43,6 +43,10 @@ To see optional CLI arguments run:
 uv run pytest --help
 ```
 
+### Model Registry upstream tests
+Be aware that it is possible to also run upstream Model Registry tests from this repository.
```

Comment:
Why do we want to run these upstream tests from this repo. Would not it be good to have separate pipelines?